development/kops/.gitignore

```diff
@@ -0,0 +1 @@
+sonobuoy
```

Comment:
We can move sonobuoy somewhere in $PATH instead.

Reply:
For now, we are going to put it right here. There is another ticket about maybe putting sonobuoy in S3, but even then, we'll just be downloading it from s3. It doesn't take long. It isn't something you install with apt-get. It is  expected to be used this way.